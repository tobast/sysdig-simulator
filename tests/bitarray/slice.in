4
11110000
10101010
00010000
10010000
