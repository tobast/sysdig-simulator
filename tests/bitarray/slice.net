INPUT a
OUTPUT z
VAR
	a:8, z:4
IN
z = SLICE 0 3 a
