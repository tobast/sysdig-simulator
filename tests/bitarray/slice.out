1111
1010
0001
1001
